src/client/routes/Labs/index.tsx

```diff
@@ -350,7 +355,17 @@ export const Labs = () => {
                 {
                   key: 'apyData',
                   header: t('components.list-card.apy'),
-                  format: ({ apyData }) => formatPercent(apyData, 2),
+                  transform: ({ apyData, apyMetadata, address }) =>
+                    !apyMetadata ? (
+                      <span>{formatPercent(apyData, 2)}</span>
+                    ) : (
+                      <Tooltip
+                        placement="bottom"
+                        tooltipComponent={<ApyTooltipData apy={apyMetadata} address={address} />}
+                      >
+                        <StyledHelperCursor>{formatPercent(apyData, 2)}</StyledHelperCursor>
+                      </Tooltip>
+                    ),
```

Comment:
we should also make a wrapper component for this one, since it repeats on many places, and will be more difficult to refactor in all of them when a change is required.